test/serverside.js

```diff
@@ -1963,6 +1963,46 @@ describe('Channel types', function() {
 				descending: true,
 			});
 		});
+
```

Comment:
a nice test is copy the default messaging permissions to a new custom channel type and expect the same behaviour

Reply:
the test is validates exactly it should: when roles are empty it should fail, when owner is present it's ok